package-lock.json

```diff
@@ -0,0 +1,1765 @@
+{
+  "name": "jsgreat",
+  "version": "0.1.9",
```

Comment:
I wonder if there is any consequence when there is a version mismatch between package.json and package-lock.json